### DISCUSSION TITLE
大佬 3.7的有提示受损

### DISCUSSION THREAD

**Discussion by aixed** [huiyadanli/RevokeMsgPatcher#358] [General]
有解决办法吗？
可以一起研究下

**Comment by aixed** [comment]
show_verify_error
85C0           test eax,eax
75 59          jnz short WeChat_-.00E66B10  //改为 jmp
试试这个看可以吗

**Comment by huiyadanli** [comment]
#345 已经加上去校验的功能了

但是微信官方在2.7.0.27+已经去掉这个校验了。。。一天一个新版本，可以看下最近的几个2.7版本的几个issue

**Comment by aixed** [comment]
ok




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年6月2日(星期四) 中午1:44
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [huiyadanli/RevokeMsgPatcher] 大佬 3.7的有提示受损 (Discussion #358)





 
#345 已经加上去校验的功能了
 
但是微信官方在2.7.0.27+已经去掉这个校验了。。。一天一个新版本，可以看下最近的几个2.7版本的几个issue
